pyuploadtool/changelog/commit.py

```diff
@@ -0,0 +1,33 @@
+from github.Commit import Commit
+
+from .author import Author
+
+
+class ChangelogCommit:
```

Comment:
This likely wants to be a
```python
class ChangelogCommit(NamedTuple):
    author: Author
    message: str
    sha: str
```

You don't need to "write-protect" attributes in Python, really.

Reply:
Oh! That's an awesome trick. Thanks!